### PR TITLE
fix(clerk-js): Align <OrganizationProfile/> with designs

### PR DESCRIPTION
**Pull request by octoper**
## Checklist

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [ ] `@clerk/backend`
- [ ] `@clerk/chrome-extension`
- [ ] `@clerk/clerk-js`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/fastify`
- [ ] `gatsby-plugin-clerk`
- [ ] `@clerk/localizations`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/remix`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/themes`
- [ ] `@clerk/types`
- [ ] `build/tooling/chore`

### CODE REVIEW

**Comment by octoper** [comment]
!preview

**Comment by clerk-cookie** [comment]
Hey @octoper, your preview is currently deploying and should be available shortly.

| Status | Preview | Updated (UTC) |
| :-- | :-- | :-- |
| :cook: _Deploying..._ | [Inspect](https://javascript-preview-fra26dlpw.clerkstage.dev) | Jan 16, 2024 11:54 PM |

**Comment by desiprisg** [comment]
!preview

File: packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationGeneralPage.tsx [hunk 1]
```diff
@@ -98,7 +99,10 @@ const OrganizationProfileSection = () => {
           fallback={profile}
         >
           <Action.Closed value='edit'>
-            <ProfileSection.Item id='organizationProfile'>
+            <ProfileSection.Item
+              id='organizationProfile'
+              sx={t => ({ maxHeight: t.space.$8 })}
```
Comment (desiprisg) [inline packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationGeneralPage.tsx:104]: Why is this needed?

File: packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationMembersTabInvitations.tsx [hunk 2]
```diff
@@ -24,41 +25,59 @@ export const OrganizationMembersTabInvitations = () => {
...
         <Protect permission={'org:sys_domains:manage'}>
-          <Col
-            gap={2}
-            sx={{
+          <Flex
+            sx={t => ({
               width: '100%',
-            }}
+              gap: t.space.$8,
+              paddingBottom: `${t.space.$4}`,
```
Comment (desiprisg) [inline packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationMembersTabInvitations.tsx:32]: No need for backticks here.

File: packages/clerk-js/src/ui/elements/Navbar.tsx [hunk 3]
```diff
@@ -261,12 +261,17 @@ const NavButton = (props: NavButtonProps) => {
...
+      hoverAsFocus
       {...rest}
       sx={[
         t => ({
           gap: t.space.$3,
           justifyContent: 'flex-start',
+          backgroundColor: isActive ? t.colors.$blackAlpha100 : undefined,
           opacity: isActive ? 1 : 0.6,
+          ':focus': {
+            boxShadow: 'none',
```
Comment (desiprisg) [inline packages/clerk-js/src/ui/elements/Navbar.tsx:273]: We can do `focusRing={false}` instead of this.

File: packages/clerk-js/src/ui/elements/Menu.tsx [hunk 4]
```diff
@@ -194,6 +194,9 @@ export const MenuItem = (props: MenuItemProps) => {
           justifyContent: 'start',
           borderRadius: theme.radii.$sm,
           padding: `${theme.space.$1} ${theme.space.$3}`,
+          ':focus': {
+            boxShadow: 'none',
```
Comment (desiprisg) [inline packages/clerk-js/src/ui/elements/Menu.tsx:198]: We can do focusRing={false} instead of this.

File: packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationMembersTabInvitations.tsx (hunk 2 above)
Comment (octoper) [reply packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationMembersTabInvitations.tsx]: Fixed!

File: packages/clerk-js/src/ui/elements/Menu.tsx (hunk 4 above)
Comment (octoper) [reply packages/clerk-js/src/ui/elements/Menu.tsx]: Fixed!

**Comment by octoper** [comment]
!preview

File: packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationGeneralPage.tsx (hunk 1 above)
Comment (octoper) [reply packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationGeneralPage.tsx:104]: This is mainly to avoid having `Organization Profile` text centered, as it solves the layout shift when the action card is open, and also it's a bit closer to the design.

**Review by anagstef — APPROVED** [review]

**Comment by octoper** [comment]
!preview
